6/main.js

```diff
@@ -0,0 +1,65 @@
+function thesmallestnumber(num1, num2, num3, num4) {
+  if (isNaN(Math.min.apply(null, [num1, num2, num3, num4]))) {
+    return "Введіть хоча б 4 числа.";
+  } else {
+    return Math.min.apply(null, [num1, num2, num3, num4]);
+  }
+}
+console.log(thesmallestnumber());
```

Comment:
What if call like
`thesmallestnumber('0', '0sdf', '0sdf', '-1')` ?

The result will be invalid